meshapp/src/theme/MainTheme.tsx

```diff
@@ -0,0 +1,229 @@
+          },
+          messageText: {
+            main: "#F1E8DF",
+          },
+          footerLogoColor: {
+            main: "#26383A",
+          },
+          mailIconColor: {
+            main: "#247C67",
+          },
+        }),
+    //universal
+    avatar: {
+      fill: "#BDBDBD",
+    },
+    chip: {
+      defaultCloseFill: "#000000",
+    },
+    error: {
+      main: "#D32F2F",
+    },
+    alert: {
+      errorContent: "#5F2120",
+      errorFill: "#FDEDED",
+    },
+    notiColor: {
+      main: "#DE8888",
+    },
+    messageNotiColor: {
+      main: "#CE808B",
+    },
+    sendMessageBackground: {
+      main: "#D9D9D966",
+    },
+    sendMessageBorder: {
+      main: "#000000CC",
+    },
+  };
+}
+
+//returns custom component theme based on dark or light mode
+function getComponents(mode: PaletteMode) {
+  //light mode
+  return {
+    ...(mode === "light"
+      ? {
+          MuiButton: {
+            styleOverrides: {
+              root: {
+                "&.MuiButton-contained:not(.Mui-disabled)": {
+                  backgroundColor: "#74D194",
+                },
+                "&.MuiButton-outlined": {
+                  borderColor: "#74D19480",
+                  color: "#247C67",
+                },
+              },
+            },
+          },
+          MuiCheckbox: {
+            styleOverrides: {
+              root: {
+                "&.Mui-checked": {
+                  color: "#74D194",
+                },
+              },
+            },
+          },
+          MuiStepIcon: {
+            styleOverrides: {
+              root: {
+                "&.Mui-completed,&.Mui-active": {
+                  color: "#74D194",
+                },
+              },
+            },
+          },
+        }
+      : {
+          //dark mode
+
+          MuiButton: {
+            styleOverrides: {
+              root: {
+                "&.MuiButton-contained:not(.Mui-disabled)": {
+                  backgroundColor: "#247C67",
+                },
+                "&.MuiButton-outlined": {
+                  borderColor: "#247C6780",
+                  color: "#27383A",
+                },
+              },
+            },
+          },
+          MuiCheckbox: {
+            styleOverrides: {
+              root: {
+                "&.Mui-checked": {
+                  color: "#247C67",
+                },
+              },
+            },
+          },
+          MuiStepIcon: {
+            styleOverrides: {
+              root: {
+                "&.Mui-completed,&.Mui-active": {
+                  color: "#247C67",
+                },
+              },
+            },
+          },
+        }),
+
+    //universal
+    MuiAlert: {
+      styleOverrides: {
+        root: {
+          "&.MuiAlert-standardError": {
+            backgroundColor: "#FDEDED",
+            color: "#5F2120",
+          },
+        },
+      },
+    },
+  };
+}
+
+const theme = createTheme({
+  ...getDesignTokens("dark"),
+});
```

Comment:
Include comment here to indicate you can swap b/w light and dark mode (for future tasks that works with it)